codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -57,6 +59,7 @@
     private final Http2FrameReader frameReader;
     private Http2FrameListener listener;
     private final Http2PromisedRequestVerifier requestVerifier;
+    private final ArrayDeque<Http2Settings> outstandingRemoteSettingsQueue;
```

Comment:
nit: Deque ?

Reply:
done

Reply:
ups sorry I think this can even be `Queue`

Reply:
+1 for `Queue` on the declared type